torchvision/models/vision_transformer.py

```diff
@@ -318,20 +318,16 @@ def _vision_transformer(
     return model
 
 
-_COMMON_META = {
+_COMMON_META: Dict[str, Any] = {
```

Comment:
I assume mypy is to blame for this, right?

Reply:
Yes, correct